doc/source/readers.rst

```diff
@@ -141,46 +141,83 @@ time etc. The following attributes are standardized across all readers:
-      confined area in which the satellite is actively maintained in using maneuvres). Inbetween
-      major maneuvres, when the satellite is permanently moved, the nominal position is constant.
-    * ``nadir_longitude/latitude``: Intersection of the instrument's Nadir with the surface of the
-      earth. May differ from the actual satellite position, if the instrument is pointing slightly
-      off the axis (satellite, earth-center). If available, this should be used to compute viewing
-      angles etc. Otherwise, use the actual satellite position.
-    * ``projection_longitude/latitude/altitude``: Projection center of the re-projected data. This
-      should be used to compute lat/lon coordinates. Note that the projection center can differ
-      considerably from the actual satellite position. For example MSG-1 was at times positioned
-      at 3.4 degrees west, while the image data was re-projected to 0 degrees.
-    * [DEPRECATED] ``satellite_longitude/latitude/altitude``: Current position of the satellite at
-      the time of observation in geodetic coordinates.
-
-    .. note:: Longitudes and latitudes are given in degrees, altitude in meters. For use in
-              pyorbital, the altitude has to be converted to kilometers, see for example
-              :func:`pyorbital.orbital.get_observer_look`.
-
-  * For *polar orbiting* satellites the readers usually provide coordinates and viewing angles of
-    the swath as ancillary datasets. Additional metadata related to the satellite position include:
-
-      * ``tle``: Two-Line Element (TLE) set used to compute the satellite's orbit
-
+  See the :ref:`orbital_parameters` section below for more information.
 * ``raw_metadata``: Raw, unprocessed metadata from the reader.
 
 Note that the above attributes are not necessarily available for each dataset.
 
+.. _time_metadata:
+
+Time Metadata
+-------------
+
+In addition to the generic ``start_time`` and ``end_time`` pieces of metadata
+there are other time fields that may be provided if the reader supports them.
+These items include:
+
+* ``observation_start_time``: The point in time when a sensor began recording
+  for the current data.
+* ``observation_end_time``: Same as ``observation_start_time``, but when data
+  has stopped being recorded.
+* ``scheduled_start_time``: The "human friendly" time describing the start of
+  the data observation interval or repeat cycle. This time is often on a round
+  minute (seconds=0). Along with the scheduled end time, these times define the
+  regular interval of the data collection. For example, GOES-16 ABI full disk
+  images are collected every 10 minutes (in the common configuration) so
+  ``scheduled_start_time`` and ``scheduled_end_time`` would be 10 minutes apart
+  regardless of when the instrument recorded data inside that interval.
+* ``scheduled_end_time``: Same as ``scheduled_start_time``, but the end of the
+  interval.
```

Comment:
Like @ameraner said in [here](https://github.com/pytroll/satpy/issues/2012#issuecomment-1033077346), these should be `repeat_cycle_{start,end}_time`, or similar.

Reply:
Are you expressing your opinion or are you telling me I did it wrong? In that discussion Andrea said that "scheduled" would be an acceptable term and based on other discussions (including in that thread) "scheduled" seemed like the more accepted term.

Reply:
Also it was pointed out that the "planned time" isn't actually available in the SEVIRI files so there wasn't a real need to have a planned versus scheduled distinction.

Reply:
That's my opinion, and also what I read @ameraner meant. In the end he was okay with these terms if the explanations didn't make sense, but the differences between "scheduled" and "repeat cycle"/"slot" will still be there. But yeah, just my opinion on the terminology :-)